examples/typescript/index.ts

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+
+const app: import("express").Application = module.exports = express();
+
+app.get('/', function(request, response) {
+    console.log(request.url);
```

Comment:
Hi @Segmentational, can you use the types here? like Request and Response types, also the import `import express from 'express';` instead of the require function.

Reply:
Absolutely, I can clean this up!